benchclients/python/tests/test_conbench_client.py

```diff
@@ -36,21 +39,13 @@ def test_cc_get_qparm(httpserver: HTTPServer):
     assert c.get("/bonjour", params={"whats": "up"}) == [2]
 
 
-def test_cc_get_500(httpserver: HTTPServer):
-    set_cb_base_url(httpserver)
-    c = ConbenchClient()
-    httpserver.expect_request("/api/foobar").respond_with_response(Response(500))
-    with pytest.raises(requests.exceptions.HTTPError, match="500 Server Error"):
-        assert c.get("/foobar") == [1, 2]
-
-
 @pytest.mark.parametrize("respjson", [[1, 2], {"1": "2"}])
 def test_cc_post(httpserver: HTTPServer, respjson):
     set_cb_base_url(httpserver)
     c = ConbenchClient()
     httpserver.expect_ordered_request(
         "/api/foobar", method="POST", json={"ql": "biz"}
-    ).respond_with_json(respjson)
+    ).respond_with_json(respjson, status=201)
```

Comment:
this adjustment is expected and good, thew client expects 201 which is what Conbench emits.